FrontEndApp/package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do we really need this lock file? 
(I don't think so)
If not, please delete this as well